bcrypt.go

```diff
@@ -179,3 +189,9 @@ func MatchBytes(password []byte, hash []byte) bool {
 	}
 	return subtle.ConstantTimeCompare(h, hash) == 1
 }
+
+func clear(w []byte) {
+	for k := range w {
+		w[k] = 0x00
+	}
+}
```

Comment:
```suggestion
		w[k] = 0
	}
}
```
GitHub complains about this missing newline - not super sure if this suggestion will fix it

Reply:
I added the newline from the editor